docs/features/models/ner.rst

```diff
@@ -41,6 +41,9 @@ Here is the list of all available configs:
     +------------------------------------------------------------------------+--------------------+          +-----------------+------------+------------+
     | :config:`ner_conll2003_bert <ner/ner_conll2003_bert.json>`             | CoNLL-2003         |          | 400 MB          |   1.3 GB   |   91.7     |
     +------------------------------------------------------------------------+--------------------+----------+-----------------+------------+------------+
+    | :config:`ner_mbert_dream_distil <ner/ner_mbert_dream_distil.json>`     | CoNLL-2003         | En+Ru    | 700 MB          |   1.6 GB   |   89.4     |
```

Comment:
Rename config according naming agreement. I'm not sure about writing both dataset names like `conll2003_collection`, but `dream` should be removed for sure, and maybe `distil` replaced with `distilbert`. Don't forget to rename model weight file and md5 file.

Reply:
@dilyararimovna how do you think we should name the config?